docs/user-documentation/datastreams.md

```diff
@@ -1,23 +1,45 @@
-## Media in Islandora 8
+Drupal 8 recognizes files (such as images, audio files, video files, etc.) but wraps each file in an intermediate structure called a
+"media" to allow us to attach fields to files. It is in a media's fields that we store information about the media's file, such as file
+size, width and height (for images), alt text (for images), creation date, and so on.
 
-Islandora 8's equivalent of Islandora 7.x datastreams are "media". Drupal 8 recognizes files (such as images, audio files, video files, etc.) but wraps each file in an intermediate structure called a "media" to allow us to attach fields (including Drupal tags) to files. It is in a media's fields that we store information about the media's file, such as file size, width and height (for images), alt text (for images), creation date, and so on.
+!!! Note
```

Comment:
Just noting, is the `!!!` supposed to do some formatting? Because it isn't and then the word `Note` doesn't have a colon after it, so you get
> !!! Note In Islandora 7

instead of something like
> **Note:** In Islandora 7

There are several of these below. 

Also this looks to be a bulleted list in Markdown, but ends up as a single paragraph when viewed. Should probably be

```
**Note:** 

* In Islandora 7, this sort of technical metadata would have been stored in a single RELS-INT datastream.
* In Islandora 8, each datastream holds its own technical metadata using media entities.
```

>**Note:** 
>
>* In Islandora 7, this sort of technical metadata would have been stored in a single RELS-INT datastream.
>* In Islandora 8, each datastream holds its own technical metadata using media entities.